uniffi_bindgen/src/interface/mod.rs

```diff
@@ -100,23 +100,6 @@ impl<'ci> ComponentInterface {
         Ok(ci)
     }
 
-    pub fn from_bincode(data: &[u8]) -> Result<Self> {
```

Comment:
The `.idl` file is the source of truth now.

Reply:
😢 I really liked this neat hack, but yeah, using the `.idl` as the source of truth makes more sense for our setup, at least in the short term.